scripts/spack/devtools_configs/toss_4_x86_64_ib/spack.yaml

```
@@ -12,7 +12,7 @@ spack:
   view:
     default:
       root: ../view
-      select: [llvm, cppcheck, doxygen, py-ats, py-sphinx]
+      select: [llvm, cppcheck, doxygen, py-ats, py-sphinx, python]
```

I found i also needed this when I ran into it on Axom
```suggestion
      select: [llvm, cppcheck, doxygen, py-ats, py-sphinx, python, ^python]
```